test/ava-files.js

```diff
@@ -25,7 +25,7 @@ test('ignores relativeness in patterns', t => {
 });
 
 test('testMatcher', t => {
```

Comment:
Should there be new test added instead of modifying existing test? That is to ensure that existing behaviour is preserved and not accidentally broken.

Reply:
Agreed, I will create a separate test case for this.

Reply:
Done